### PR TITLE
fix: Improve access to label list in Text Classification

### LINKED ISSUE #1804: UI: Text Classification - Record card: improve access to labels list
**Describe the bug**
When label list is long, there is no way to show all the available labels without having to click the + sign on each records

**To Reproduce**
1. Go to my dataset  in annotation mode
2. Click on "+ sign" to view the full label list
3. Have to click again on the + sign on each records

**Expected behavior**
"it’s that +4 that I wanted to get rid of and instead just show the 4 labels missing".
View directly the full list

**Screenshots**

**Environment (please complete the following information):**
 - OS [e.g. iOS]:
 - Browser [e.g. chrome, safari]:
 - Argilla Version [e.g. 1.0.0]:
 - ElasticSearch Version [e.g. 7.10.2]:
 - Docker Image (optional) [e.g. argilla:v1.0.0]:

**Additional context**
"We have around 25~30 labels"

"we reduced it to around 15 but still we can’t see all the options, and the annotators always have to click the + to see the extra labels"

### PR DESCRIPTION
**Pull request by leiyre**
close #1804

This PR allows to see all labels when pagination size is 1 record in hand labeling view for Text Classifier.

### CODE REVIEW

File: frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue [hunk 1]
```diff
@@ -93,6 +93,11 @@ export default {
         this.selectedLabels = this.appliedLabels;
       }
     },
+    allowToShowAllLabels(n, o) {
+      if (n !== o) {
+        this.shownLabels = this.maxVisibleLabels;
+      }
```
Comment (keithCuniah) [inline frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue]: n and o are not enought explicit, it's better to put words

File: frontend/components/base/BasePagination.vue [hunk 2]
```diff
@@ -195,13 +195,10 @@ export default {
...
-        if (elem === document.activeElement) {
-          if (event.keyCode === 39) {
-            this.nextPage();
-          } else if (event.keyCode === 37) {
-            this.prevPage();
-          }
+        if (event.keyCode === 39) {
+          this.nextPage();
+        } else if (event.keyCode === 37) {
+          this.prevPage();
```
Comment (keithCuniah) [inline frontend/components/base/BasePagination.vue:203]: you can be more explicit by putting constants for the keycode for arrowRight and arrowLeft. For later to know what means these code

File: frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue (hunk 1 above)
Comment (leiyre) [reply frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue]: I have removed the watcher, we don't need it

File: frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue [hunk 3]
```diff
@@ -125,6 +126,37 @@ export default {
     isMultiLabel() {
       return this.dataset.isMultiLabel;
     },
+    visibleLabels() {
+      const selectedLabels = this.filteredLabels.filter(
+        (l) => l.selected
+      ).length;
```
Comment (keithCuniah) [inline frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue:136]: rename selectedLabels by numberOfSelectedLabels to be more explicit

**Review by keithCuniah — APPROVED** [review]

**Pull request merged**